medusa/metadata/kodi_12plus.py

```diff
@@ -233,21 +224,24 @@ def _ep_data(self, ep_obj):
             try:
                 my_ep = my_show[ep_to_write.season][ep_to_write.episode]
             except (IndexerEpisodeNotFound, IndexerSeasonNotFound):
-                logger.log(u'Unable to find episode {ep_num} on {indexer}... '
-                           u'has it been removed? Should I delete from db?'.format
-                           (ep_num=episode_num(ep_to_write.season, ep_to_write.episode),
-                            indexer=indexerApi(ep_obj.show.indexer).name))
+                log.info(
+                    u'Unable to find episode {ep_num} on {indexer}...'
+                    u' has it been removed? Should I delete from db?', {
+                        'ep_num': episode_num(ep_to_write.season, ep_to_write.episode),
+                        'indexer': indexerApi(ep_obj.show.indexer).name,
+                    }
+                )
                 return None
 
             if not getattr(my_ep, 'firstaired', None):
                 my_ep['firstaired'] = str(datetime.date.fromordinal(1))
 
             if not getattr(my_ep, 'episodename', None):
-                logger.log(u'Not generating nfo because the ep has no title', logger.DEBUG)
+                log.debug(u'Not generating nfo because the ep has no title')
                 return None
 
-            logger.log(u'Creating metadata for episode {ep_num}'.format
-                       (ep_num=episode_num(ep_obj.season, ep_obj.episode)), logger.DEBUG)
+            log.debug(u'Creating metadata for episode {0}',
```

Comment:
I don't see anything wrong with positional arguments in cases like this where it doesn't impact the clarity of the message when read.